configs/infer_config.yaml

```diff
@@ -40,3 +40,7 @@ batches_per_worker: 16
 result_fname: inferred
 gen_bigwig: True
 deletebg: False
+
+#Eval options
+best_metric_choice: "AUROC"
```

Comment:
Why do we need best_metric during inference?

Reply:
We need for evaluation. Not for inference. `python main.py eval` needs it. This is in addition to all the options required by infer. It didn't make sense to have yet another config file, so I added these options in the infer_config.